components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -682,6 +684,91 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
          endif
 
          nEdges = nEdgesArray(3)
+         if (config_GM_horizontal_taper == 'RossbyRadius'.or. &
+             config_Redi_horizontal_taper == 'RossbyRadius') then
+            allocate (RossbyRadiusTaper(nEdges))
+            call mpas_pool_get_array(meshPool, 'fEdge', fEdge)
+
+            !$omp parallel
+            !$omp do schedule(runtime) private(k, cell1, cell2, sum_hN, lt1, lt2, c_mode1, resolution)
+            do iEdge = 1, nEdges
+               cell1 = cellsOnEdge(1, iEdge)
+               cell2 = cellsOnEdge(2, iEdge)
+               sum_hN = 0.0
+
+               do k = minLevelEdgeBot(iEdge)+1, maxLevelEdgeTop(iEdge)-1
+
+                  lt1 = 0.5_RKIND*(layerThickness(k,cell1) + layerThickness(k-1,cell1))
+                  lt2 = 0.5_RKIND*(layerThickness(k,cell2) + layerThickness(k-1,cell2))
+                  sum_hN = sum_hN + 0.5_RKIND*(lt1*sqrt(max(BruntVaisalaFreqTop(k,cell1),0.0_RKIND)) + &
+                                  lt2*sqrt(max(BruntVaisalaFreqTop(k,cell2),0.0_RKIND)))
```

Comment:
I have to say, it's pretty insane that we have lived for 9 years with a variable named `BruntVaisalaFreqTop` that is actually the Brunt Vaisala frequency *squared*.  That isn't good.

Reply:
To be fair...in most practical instances, it's always squared since we use it for stability and not to compute an actual frequency. So while it may be important for units, I'm not sure there has been any harm due to a laxness in language.

Reply:
It did lead to a confusing conversation on this PR and then quite a bit of confusion when I was looking at the code.  So I think it's just pretty bad for code readability.

Reply:
If the variable were `BruntVaisalaFreqSqrTop` instead, it would only be 3 characters longer and would avoid this silly confusion.

Reply:
Well, I'm not a fan of SelfDescribingVariableNamesThatConsumeAnEntireLineOfCode anyway, so BVSqTop is fine with me :^)

Reply:
Yeah, previously I was okay with `SelfDescribingVariableNamesThatConsumeAnEntireLineOfCode` because they're easy to copy and paste.  But they sure don't play nice with short terminal lines and line-length limits.  I'd be happy with `BVSqTop`. At least it's not `N2`...